Doc/library/collections.abc.rst

```diff
@@ -28,81 +28,80 @@ An :func:`issubclass` or :func:`isinstance` test for an interface works in one
 of three ways.
 
 1) A newly written class can inherit directly from one of the
-abstract base classes.  The class must supply the required abstract
-methods.  The remaining mixin methods come from inheritance and can be
-overridden if desired.  Other methods may be added as needed:
+   abstract base classes.  The class must supply the required abstract
+   methods.  The remaining mixin methods come from inheritance and can be
+   overridden if desired.  Other methods may be added as needed:
 
-.. testcode::
+   .. testcode::
 
-    class C(Sequence):                      # Direct inheritance
-        def __init__(self): ...             # Extra method not required by the ABC
-        def __getitem__(self, index):  ...  # Required abstract method
-        def __len__(self):  ...             # Required abstract method
-        def count(self, value): ...         # Optionally override a mixin method
+       class C(Sequence):                      # Direct inheritance
```

Comment:
Shouldn't this be aligned with the `t` of `testcode`?

Reply:
It wasn't before (I just prepended 3 spaces). All that really matters is that the indentation is consistent, as common leading whitespace is stripped. If you'd prefer I change for style, though, I can.

Reply:
I think we usually align the code with `testcode` and `codeblock` directives, although we then need 4 spaces in the actual code (so it's a 7 indent in general, and it's not the best for IDEs, but it's usually better visually when reading the rST).

Since we did this for

```rst
.. testsetup:: *

   from collections.abc import *
   import itertools
__name__ = '<doctest>'
```

let's be consistent here as well.